crates/teloxide/src/repls/commands_repl.rs

```diff
@@ -1,3 +1,8 @@
+use std::fmt::Debug;
```

Comment:
?

Reply:
Initially I wanted just to add `<B as Requester>::Err` bound to the `Polling` and similar things. But such approach requires to change to much code (particularly in the `command_repl`).

Just noticed that I can reorder imports nicely here according to our `CODE_STYLE.md`